domains/aryanmaheshwari.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Aryan2602",
+        "email":"aryan26.03.02@gmail.com"
+    },
+    "records": {
+      "CNAME": "https://aryan-2602.github.io/personal-website/"
```

CNAME doesn't allow trailing (/) and https://
```suggestion
      "CNAME": "aryan-2602.github.io"
```